src/components/screens/IndexScreen/IndexScreen.js

```diff
@@ -240,14 +319,82 @@ export function PureIndexScreen({ ...props }) {
         logo={GitlabLogoSVG}
       />
 
-      <CTA
-        text={<span>Storybook is quick to install and it’s easy to get started.</span>}
-        action={
-          <Button appearance="secondary" isLink href={docs}>
-            Go to the docs
-          </Button>
-        }
-      />
+      <Contrast>
+        <Separator />
+        <ValueProp
+          orientation="right"
+          media={
+            <Placeholder ratio={0.75}>
+              <Video
+                src="videos/storybook-publish-review-greybg-optimized-lg.mp4"
+                alt="Publish to review workflow video"
+                shouldChangeSize
+              />
+            </Placeholder>
+          }
+          title="Share to get sign off from teammates"
+          desc="Stories show how UIs actually work not just how they're supposed to work. That makes
+            gathering feedback and reproductions easy."
+          lazyloadPlaceholder={<Placeholder ratio={0.75} />}
+        />
+        <BenefitList>
+          <BenefitItem
+            image={<img src="/images/home/share-publish.png" alt="Publish Storybook online" />}
+            title="Publish to get feedback"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/publish-storybook`}>Publish Storybook</Link>{' '}
+                online to give your team a universal reference point for feedback.
+              </>
+            }
+          />
+          <BenefitItem
+            image={<img src="/images/home/share-embed.png" alt="Embed stories" />}
+            title="Embed stories in your favorite sites"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/embed`}>Embed stories</Link> to showcase your work
+                to teammates and the developer community. Works with the oEmbed standard.
+              </>
+            }
+          />
+          <BenefitItem
+            image={<img src="/images/home/share-compose-storybooks.png" alt="Compose Storybooks" />}
+            title="Compose multiple Storybooks together"
+            desc={
+              <>
+                <Link href={`${docs}react/sharing/composition`}>Reference external Storybooks</Link>{' '}
+                inside your local Storybook, no matter the view layer, tech stack, or dependencies.
+              </>
+            }
+          />
+          <BenefitItem
+            image={<img src="/images/home/share-import-stories.png" alt="Import stories" />}
+            title={
+              <>
+                <Code>import</Code> stories into other JavaScript tooling
+              </>
+            }
+            desc={
+              <>
+                Stories are a <Link href={`${docs}react/api/csf`}>portable standard</Link> based on
```

Comment:
Would this be a better place to link to?

```suggestion
                Stories are a <Link href={`${docs}react/writing-tests/importing-stories-in-tests`}>portable standard</Link> based on
```